File: Java/Disk.java
class Disk {

    private String type;
    private String capacity;
    private int price;

    Disk(){ }

    Disk(String t, String c, int p){
        this.type = t;
        this.setCapacity(c);
        this.price = p;
    }

    void setType(String t){ this.type = t; }
    String getType(){ return this.type; }
    
    public String getCapacity() {
        return capacity;
    }
    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }
    void printDetDisk(){
        System.out.println("----------------------");
        System.out.println("Disk Type      : " + this.type);
        System.out.println("Disk Capacity  : " + this.capacity);
        System.out.println("Price          : " + this.price);
    }
}

File: Python/Disk.py
class Disk:

    def __init__(self, tipe, capacity, price):
        self.type = tipe
        self.capacity = capacity
        self.price = price

    def setType(self, tipe):
        self.type = tipe
    def getType(self):
        return self.type

    def setCapacity(self, capacity):
        self.capacity = capacity
    def getCapacity(self):
        return self.capacity

    def setPrice(self, price):
        self.price = price
    def getPrice(self):
        return self.price

    def printDetDisk(self):
        print("----------------------")
        print("Disk Type      : ", self.type)
        print("Disk Name      : ", self.capacity)
        print("Price          : ", self.price)
